_sass/_code.scss

```diff
@@ -0,0 +1,24 @@
+.code {
+  background-color: $code-background-color;
+  border-radius: $base-border-radius;
```

Comment:
Properties should be ordered background-color, border, border-radius, font-size, margin-bottom